sycl/test-e2e/Graph/Explicit/add_nodes_after_finalize.cpp

```diff
@@ -1,10 +1,13 @@
 // REQUIRES: level_zero, gpu
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
+// RUN: env ZE_DEBUG=4 %{run} %t.out
 
 // This test adds a new node with the explicit API to an already finalized
 // modifiable graph, before finalizing and executing the graph for a second
 // time.
+// The second run is to check that there are no leaks reported with the embedded
+// ZE_DEBUG=4 testing capability.
```

Comment:
I'm curious what kind of failure message this produces from `llvm-lit` when a leak is detected? Since we're not checking the output to verify there are no leaks.

Reply:
Good point.
The previous version of runtime throws a PI_ERROR_INVALID_MEM_OBJECT exception.
So the test fails with this exception and the leaks status is displayed.
But the new one (after UR merge) throws no exceptions.
So I think I need to port this exception throwing to this branch.

Reply:
To not modify the exception policy of UR, I finally added a llvm-lit test check in all tests.
This test is based on the ZE_DEBUG summary display and raises an error if a leak is detected by the runtime support.